proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,196 @@
+# Title
+
+- Author Name [Kat Marchan (@zkat)](https://github.com/zkat)
+- Start Date 2021-03-04
+- GitHub Issue (GitHub Issue link)
+- GitHub PR (GitHub PR link)
+
+## Summary
+
+This specifies the syntax and behavior for the `dotnet.exe nuget trusted-signers` command.
+
+## Motivation
+
+This is part of a larger effort to reach full parity between `nuget.exe` and
+`dotnet.exe` for NuGet-related operations.
+
+## Explanation
+
+### Functional explanation
+
+#### `dotnet nuget trusted-signers`
+
+```
+Provides the ability to manage the list of trusted signers.
+
+USAGE:
+    dotnet nuget trusted-signer [OPTIONS] [COMMAND]
+
+COMMANDS:
+    list (default)
+    trust-package <name> <package>...
+    trust-source <name> [source-url]
+    trust-certificate <name> <fingerprint>
+    remove <name>
+    sync <name>
+```
+
+#### `dotnet nuget trusted-signers list`
+
+```
+Lists all the trusted signers in the configuration. This option will include all the certificates (with fingerprint and fingerprint algorithm) each signer has. If a certificate has a preceding [U], it means that certificate entry has allowUntrustedRoot set as true.
+
+USAGE:
+    dotnet nuget trusted-signers [OPTIONS] list
+
+OPTIONS:
+    -h, --help                          Prints help information.
+    -v, --verbosity <LEVEL>             Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
```

Comment:
It looks `verbosity` thing doesn't make sense here, I don't see it in nuget.exe implementation code.

Reply:
I think that make sense.
We don't have verbosity options in [dotnet nuget add source](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-nuget-add-source) either, which is also just editing the config file.

Reply:
nuget.exe supports verbosity on every command. 
I think we should lean towards supporting verbosity for most commands in dotnet.exe as well.

Reply:
Thanks @nkolev92! Good to know the future trends.

Reply:
There's a set of options that should be available in all commands. This is one of them, whether we take advantage of them or not.